### PR TITLE
Yael/process2metrics ga

### PR DESCRIPTION
**Pull request by shlomoartzi**
### What does this PR do?
- updates feature status to GA by removing the beta form  
- changes feature title to 'Increase Process Retention' 
- update images to reflect new feature name (feature tab is now labeled 'Increase Retention')

### Motivation
Increased process retention is GA. 

### Preview

https://docs-staging.datadoghq.com/Yael/process2metrics_ga/infrastructure/process/generate_process_metrics/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/infrastructure/process/generate_process_metrics.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-title: Generate Metrics from Processes
+title: Increase Process Retention
 kind: documentation
 aliases:
     - /processes/processing/process_to_metrics/
-    - /infrastructure/generate_process_metrics/
+    - /infrastructure/increase_process_retention/
```
Comment (jtappa) [inline content/en/infrastructure/process/generate_process_metrics.md:6]: This is referring to the URL/filename, which you've left unchanged, so it breaks the site. Either we need to change it back or completely rename the file and setup a redirect from the old one to here.

Comment (shlomoartzi) [reply content/en/infrastructure/process/generate_process_metrics.md:6]: Got it, thanks! I'll change it back for now, and would you be able to kick off the process to rename it for the future?

Comment (jtappa) [reply content/en/infrastructure/process/generate_process_metrics.md]: I will also follow up with the name change in the sidebar navigation at the same time.

Comment (shlomoartzi) [reply content/en/infrastructure/process/generate_process_metrics.md]: Thanks! Otherwise - is this good to go? (can I merge?)

**Review by jtappa — APPROVED** [review]

**Pull request merged**